src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,50 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, K> {
+    private T item1;
+    private K item2;
```

Comment:
let's not use numbers in variables names. (you can take a look a getters name and figure out how to name these fields

Reply:
done

Reply:
done